src/sas/sascalc/invariant/invariant.py

```diff
@@ -360,7 +360,7 @@ def fit(self, power=None, qmin=None, qmax=None):
             try:
                 inv_cov = np.dot(A.transpose(), A)
                 cov = np.linalg.pinv(inv_cov)
-                err_matrix = math.fabs(residuals) * cov
+                err_matrix = np.fabs(residuals) * cov
```

Comment:
There's another `math.fabs(residuals)` invocation in line 349. Should it also be replaced?

(Maybe we should standardize on replacing all `math.` with `np.`?)